assets/assets.json

```diff
@@ -252,6 +233,13 @@
 		],
 		"supportURL": "https://forums.lanik.us/"
 	},
+	"ublock-annoyances": {
+		"content": "filters",
+		"group": "social",
```

Comment:
Do we use this `group` field in Brave? In other words, is this change significant?

Reply:
I don't believe so, in brave://adblock is not ordered like uBo. Though its something we should copy (while offering other lists)